wagtail/admin/templates/wagtailadmin/bulk_actions/footer.html

```diff
@@ -6,7 +6,7 @@ <h2 id="bulk-actions-heading" class="visuallyhidden">{% trans "Bulk actions" %}<
         <ul class="bulk-actions-buttons">{% bulk_action_choices app_label model_name %}</ul>
         <span data-bulk-action-num-objects class="num-objects"></span>
         {% if objects.has_other_pages %}
-            <button type="button" data-bulk-action-num-objects-in-listing class="num-objects-in-listing u-hidden">{{ select_all_obj_text }}</button>
+            <button type="button" data-bulk-action-num-objects-in-listing class="num-objects-in-listing !w-hidden">{{ select_all_obj_text }}</button>
```

Comment:
The ones in `bulk-actions.js` do not use `!`, so we should remove it from here, otherwise the feature doesn't work.

```suggestion
            <button type="button" data-bulk-action-num-objects-in-listing class="num-objects-in-listing w-hidden">{{ select_all_obj_text }}</button>
```